### DISCUSSION TITLE
Change interface

### DISCUSSION THREAD

**Discussion by giovi321** [immich-app/immich#12946] [Q&A]
Hi, I couldn't find a way to change the binded ip address or interface on immich server.
Thanks :)

**Comment by bo0tzz** [comment]
https://immich.app/docs/install/environment-variables#ports

**Reply by giovi321** [reply]
Thank you for helping. I did try that already by stopping the container, editing .env adding the follwowing lines at the end of the file:

```
IMMICH_HOST=192.168.56.110
IMMICH_TRUSTED_PROXIES=192.168.56.105

```
but I still cannot connect to it neither directly neither through the apache2 reverse proxy

EDIT: i did restart the container with `docker compose up -d --force-recreate`

**Reply by bo0tzz** [reply]
Unless you're running a non-default setup, Immich is inside a docker container and so you probably want to do whatever you're trying with docker port binds instead.

**Reply by giovi321** [reply]
I am running a default setup with docker. I don't understand why it is listening on all interfaces and still I cannot connect to it.
```
docker-pr 39764 root    4u  IPv4 131580      0t0  TCP *:2283 (LISTEN)
docker-pr 39771 root    4u  IPv6 132194      0t0  TCP *:2283 (LISTEN)

```

**Reply by giovi321** [reply]
Ok I understood where the problem lays. The library and postgres folders are located in a virtualbox shared folder, and immich didn't have permissions to access them.
